homeassistant/components/google_assistant/smart_home.py

```diff
@@ -69,6 +73,13 @@ def deep_update(target, source):
     return target
 
 
+def get_google_type(domain, device_class):
+    """Google type based on domain and device class."""
+    if DEVICE_CLASS_TO_GOOGLE_TYPES.get(device_class) is not None:
```

Comment:
device classes are not guaranteed unique per domain, so we should make the key be `(domain, device_class)`. Note the `(`, which means it's a tuple and so it can be used as a key.

The const becomes

```python
DEVICE_CLASS_TO_GOOGLE_TYPES = {
    (cover.DOMAIN, cover.DEVICE_CLASS_GARAGE): TYPE_GARAGE,
}
```